aviary/subsystems/propulsion/hamilton_standard.py

```diff
@@ -604,20 +605,20 @@ def compute(self, inputs, outputs):
             for k in range(2, 7):
                 AF_adj_CP[k] = AF_adj_CP[1]
                 AF_adj_CT[k] = AF_adj_CT[1]
-            if (inputs['adv_ratio'][0] <= 0.5):
-                AFCTE = 2.*inputs['adv_ratio'][0] * \
+            if (inputs['adv_ratio'][i_node] <= 0.5):
```

Comment:
Right, line 585 defines the loop that increments `i_node`. Instead of vectorizing the code (messy, confusing, challenging) we naively loop over it to allow for `num_nodes>1`. Future work could vectorize it but this might be non-trivial.